src/context.cpp

```diff
@@ -1093,7 +1162,10 @@ namespace librealsense
                 if( ! sensor_info.proxy )
                 {
                     // This is a new sensor we haven't seen yet
-                    sensor_info.proxy = std::make_shared< dds_sensor_proxy >( stream->sensor_name(), this, _dds_dev );
+                    if( stream->sensor_name().compare( "RGB Camera" ) == 0 )
```

Comment:
Isn't it some member in the realdds::dds_stream class that can return the stream type (as RS2_STREAM_COLOR)?

Reply:
No.
And we can't go over all the profiles and test if there is `RGB8` one, because D405 has `RGB8` but not a `color_sensor`.

Comparing sensor name to "RGB Camera" is also the way it was done in ROS wrapper.

Reply:
ok